charts/sequencer/files/cometbft/config/genesis.json

```diff
@@ -3,13 +3,62 @@
+      },
+      "bridge_unlock": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.bridgeUnlock.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.bridgeUnlock.multiplier }}
+      },
+      "fee_asset_change": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.feeAssetChange.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.feeAssetChange.multiplier }}
+      },
+      "fee_change": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.feeChange.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.feeChange.multiplier }}
+      },
+      "ibc_relay": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ibcRelay.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ibcRelay.multiplier }}
+      },
+      "ibc_relayer_change": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ibcRelayerChange.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ibcRelayerChange.multiplier }}
+      },
+      "ibc_sudo_change": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ibcSudoChange.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ibcSudoChange.multiplier }}
+      },
+      "ics20_withdrawal": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ics20Withdrawal.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.ics20Withdrawal.multiplier }}
+      },
+      "init_bridge_account": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.initBridgeAccount.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.initBridgeAccount.multiplier }}
+      },
+      "sequence": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.sequence.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.sequence.multiplier }}
+      },
+      "sudo_address_change": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.sudoAddressChange.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.sudoAddressChange.multiplier }}
+      },
+      "transfer": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.transfer.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.transfer.multiplier }}
+      },
+      "validator_update": {
+        "base": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.validatorUpdate.base }},
+        "multiplier": {{ include "sequencer.toUint128Proto" .Values.genesis.fees.validatorUpdate.multiplier }}
+      }
```

Comment:
These changes should all be wrapped in a check for if in `.Values.global.dev` mode, which indicates we are running on latest main vs latest release.

In general we have done this format, which I know seems kinda backwards:

```
    {{- if not .Values.global.dev }}
    <anything that should be deleted after upgrade>
    {{- else }}
    <anything that should be standard after upgrade only included in dev mode right now>
    {{- end }}
```